src/config.ts

```diff
@@ -588,6 +594,12 @@ const CHAINS: Chain[] = [
         address: "0xDF9BF58Aa1A1B73F0e214d79C652a7dd37a6074e",
         fromBlock: 123566896,
       },
+      // Allo V1 -> V2 Migration
+      // {
+      //   contractName: "AlloV2/AlloV1ToV2ProfileMigration",
+      //   address: "0x",
+      //   fromBlock: 0,
+      // },
```

Comment:
^ @boudra This will be updated when we deploy it on live and hence leaving it commented out